Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R

```diff
@@ -0,0 +1,38 @@
+# setup: clone the repo RPackageValidationTutorial.04E02
+
+
+########### Task 04_E02 A
+# initiate a validation report for requirement adoption
+vt_use_report(template = "requirements")
+vt_validate_source()
+# Discussion: How is this different from the default template?
+# Discussion: How are we controlling which validation child files are included?
+# Discussion: Where does the pdf save to? how is this different than using `vt_validate_report`
+
+########### Task 04_E02 B
+# switch to full validation report
+
+# Activity: in "validation.yml"
+#    - Replace `Requirement_Adoption.Rmd` with `validation.Rmd`
+#    - Replace `Requirement_Adoption_Report_{package}_v{version}_{date}` with `Validation_Report_{package}_v{version}_{date}`
+vt_use_report()
+vt_validate_source()
```

Comment:
To me this indicates that we should be able to provide "override" abilities to vt_validate_* or provide a separate function for rendering non-validation documents. vt_render_report() or something?

Reply:
Perhaps this is getting into philosophical territory - I see the generation of requirements to be part of a validation process. From a purely technical perspective, I really do want to drive home that we are implementing a DRY system here - the requirements are a living source, but the pdfs are the snapshots that a validation process demands.

Reply:
I would agree with that - the reason we are considering partial renders are for inherently non DRY parts that an org might require - such as getting approval for requirements or test cases prior to actually executing them. An opportunity to break out of our framework that encourages DRY behavior may be a necessity

Reply:
But anyway, not something we need to resolve this moment and technically they can still just call rmarkdown::render or click "knit"